packages/devextreme/js/__internal/grids/grid_core/editing/m_editing_form_based.ts

```diff
@@ -318,7 +318,7 @@ const editingControllerExtender = (Base: ModuleType<EditingController>) => class
   }
 
   getFormEditorTemplate(cellOptions, item) {
-    const column = this.component.columnOption(item.dataField);
+    const column = this.component.columnOption(item.name || item.dataField);
```

Comment:
```suggestion
    const column = this.component.columnOption(item.name ?? item.dataField);
```

Reply:
@pomahtri, I think it's better to leave '||'.
Because in this case: '' ?? 'field', the empty string will win

Reply:
isn't it okay?

Reply:
> isn't it okay?

It seems to me that not. Because the name is an identifier and it must contain a unique value. If it is empty ('' || null || undefined) we should target the dataField.